modules/LB/LB.tf

```diff
@@ -63,9 +73,9 @@ resource "aws_lb_target_group" "TG" {
 }
 
 resource "aws_route53_record" "my_dns_record" {
-  zone_id = "Z05599622K22XV64ERVX6"
-  name    = "mentorship"
+  zone_id = data.aws_route53_zone.hosted_zone.id
```

Comment:
Could you please, change `zone_id = data.aws_route53_zone.hosted_zone.id` to `zone_id = data.aws_route53_zone.hosted_zone.zone_id`